pyproject.toml

```diff
@@ -1,33 +1,33 @@
 [tool.black]
+# Only used by `hf-doc-builder´.
 line-length = 119
-target-version = ['py36']
+target-version = ['py38']
 
 [tool.ruff]
-ignore = ["C901", "E501", "E741", "W605"]
-select = ["C", "E", "F", "I", "W"]
+target-version = "py38"
 line-length = 119
 
-[tool.ruff.isort]
+[tool.ruff.lint]
+extend-select = [
+    "C", # Complexity
+    "E", # PEP8 errors
+    "F", # PEP8 formatting
+    "I", # Import sorting
+    "UP", # Pyupgrade upgrades
+    "W", # PEP8 warnings
+]
+ignore = [
+    "C901", # Function too complex
+    "E501", # Line length (handled by ruff-format)
+    "E741", # Ambiguous variable name
+    "UP007", # X | Y style Unions
+    "W605", # Invalid escape sequence
```

Comment:
What is the reasoning for ignoring this?

Reply:
It was ignored in the preimage. I'll check if any of the original ignores can be safely removed.

Reply:
Okay, E741 and W605 were easy enough to fix, so did that.